react/week1/todo-app/src/activities.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+function HeaderTitle() {
+  return (
+    <>
+      <h1>Todo list</h1>
+    </>
+  );
+}
+
+function ListItems(props) {
+  return (
+    <>
+      <ol>
+        <li>
+          {props.task},{props.deadlineDate}
+        </li>
+      </ol>
+    </>
+  );
+}
+
+function Activities(props) {
+  return (
+    <>
+      <HeaderTitle />
+
+      {props.TodoList.map((todo) => {
+        return <ListItems task={todo.task} deadlineDate={todo.deadlineDate} />;
```

Comment:
Seems like you are creating different orderlist <ol> for every item here. Can you try to modify the code that all the list items are under single ol.Thanks

Reply:
thanks for the feedback but Its one header with three items , its not different order list for each item ! I have the capture but couldnt post it here .